### DISCUSSION TITLE
[Unstable] Acid emitters are blocked by platforms and pass through walls

### DISCUSSION THREAD

**Discussion by zhu-rengong** [FakeFishGames/Barotrauma#14304] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

As the title said, I found this when modding some gas grenades, it makes my brain broken.

https://github.com/user-attachments/assets/f00f57dd-8efd-4667-b34f-d962d32fe8c0

https://github.com/user-attachments/assets/731c47cc-a619-451e-8463-eec0d40f8b08

https://github.com/user-attachments/assets/a5bac987-fe2e-44c2-910a-61c450728b08

### Reproduction steps

1. Throw the acid grenade a little below the platform

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.6.101.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested the fix for this.
Emitters doen't go through hull boundaries and are blocked by walls, floor, docking ports etc
Closing discussion as resolved
